src/BuildScriptGenerator/DotNetCore/DotNetCoreBashBuildSnippet.sh.tpl

```diff
@@ -93,6 +93,7 @@ fi
 mkdir -p "$DESTINATION_DIR"
 
 {{ if PreBuildCommand | IsNotBlank }}
+echo "Running the pre-build script..."
```

Comment:
You can ignore these since these will use the common constants file using the prologue and epilogue